### ISSUE TITLE
Duplicate overloads are not flagged as errors until a method call is present

### ISSUE THREAD

**Issue by FullValueRider** [WaynePhillipsEA/twinbasic#356]
**Describe the bug**
Duplicate overloaded methods can exist in code and no error messages occur.  Once a call to the overloaded methods is made the call is flagged as an error.

Demonstration Code

```
Sub Test()
    
        'Instance1  'Uncomment this line to see error
   End Sub
    
    Sub Instance1(ByVal ipValue As Long)
        
    End Sub
    
    Sub Instance1(ByVal ipvalue As Long)
        
    End Sub
```

**Expected behavior**
The error should preferably be flagged when the second method signature is created.

**Comment by WaynePhillipsEA** [comment]
Yes, that needs to be improved.  Thanks for reporting.

**Comment by Kr00l** [comment]
The same issue is when 2 enum names are the same. Probably more issues with Consts etc.

**Comment by WaynePhillipsEA** [comment]
Fixed in v0.10.4441, and will also flag duplicate components, udts and enums as well (in the same scope).  Thanks for reporting.
